pkg/apis/migration/migration.go

```diff
@@ -407,3 +417,61 @@ func (volumeMigration *volumeMigration) registerVolume(ctx context.Context, volu
 	}
 	return volumeID.Id, nil
 }
+
+// cleanupStaleCRDInstances helps in cleaning up stale volume migration CRD instances
+func (volumeMigration *volumeMigration) cleanupStaleCRDInstances(ctx context.Context) {
+	log := logger.GetLogger(ctx)
+	ticker := time.NewTicker(time.Duration(volumeMigrationInstance.cnsConfig.Global.VolumeMigrationCRCleanupIntervalInMin) * time.Minute)
+	for range ticker.C {
+		config, err := k8s.GetKubeConfig(ctx)
```

Comment:
Can we create a new context in this routine to avoid facing timed out context and track logs from each different cleanup cycle?

for example - https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/master/pkg/syncer/metadatasyncer.go#L275-L276

Reply:
Got it, thanks